demo_classification_tf.py

```diff
@@ -0,0 +1,27 @@
+import tensorflow as tf
+import numpy as np
+import cv2 as cv
+import argparse
+from classification import TensorFlowClassification
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--graph', help='.pb graph path', default='resnet_v2_101_299_frozen.pb')
+    parser.add_argument('--image', help='image path', required=True)
+    argv = parser.parse_args()
+
+    network = TensorFlowClassification(argv.graph)
+
+    with open('classification_classes_ILSVRC2012.txt', 'rt') as f:
```

Comment:
No, no, it was only about evaluation script. For demo, please keep labels file as argument